### DISCUSSION TITLE
建议增加个绑定自定义域名的功能

### DISCUSSION THREAD

**Discussion by k79e** [SeaHOH/GotoX#119] [点子]
就是那个appid使用自定义的域名.
这样就可以用appspot之外的域名了!!

2019-09-11 17:23:06
我晕 原来本身就支持的
但是这个咋用啊!!!!!

**Comment by SeaHOH** [comment]
已发布版本还未包含此功能，需手动更新至主分支。

**Comment by k79e** [comment]
我试试去!!!

**Comment by k79e** [comment]
好了有效果了!!!
